great_expectations/render/renderer/content_block/bullet_list_content_block.py

```diff
@@ -107,67 +150,243 @@ def expect_column_unique_value_count_to_be_between(cls, expectation, column_name
+            return [{
+                "template": " has a bogus $expectation_name expectation.",
+                "params": {
+                    "expectation_name": "expect_column_pair_values_A_to_be_greater_than_B"
+                }
+            }]
+
+        if params["mostly"] == None:
+            if params["or_equal"] in [None, False]:
+                return [{
+                    "template": "Values in $column_A must always be greater than those in $column_B.",
+                    "params": params
+                }]
+            else:
+                return [{
+                    "template": "Values in $column_A must always be greater than or equal to those in $column_B.",
+                    "params": params
+                }]
+
+        else:
+            if params["or_equal"] in [None, False]:
+                return [{
+                    "template": "Values in $column_A must be greater than those in $column_B at least $mostly % of the time.",
+                    "params": params
+                }]
+            else:
+                return [{
+                    "template": "Values in $column_A must be greater than or equal to those in $column_B at least $mostly % of the time.",
+                    "params": params
+                }]
+
+    @classmethod
+    def expect_column_pair_values_to_be_equal(cls, expectation):
+        params = substitute_none_for_missing(
+            expectation["kwargs"],
+            ["column_A", "column_B",
+                "ignore_row_if", "mostly", ]
+        )
+
+        # NOTE: This renderer doesn't do anything with "ignore_row_if"
+
+        if (params["column_A"] is None) or (params["column_B"] is None):
+            # FIXME: this string is wrong
+            return [{
+                "template": " has a bogus $expectation_name expectation.",
+                "params": {
+                    "expectation_name": "expect_column_pair_values_to_be_equal"
+                }
+            }]
+
+        if params["mostly"] == None:
+            return [{
+                "template": "Values in $column_A and $column_B must always be equal.",
+                "params": params
+            }]
+
+        else:
+            # Note: this pattern for type conversion seems to work reasonably well.
+            # Note: I'm not 100% sure that this is the right place to encode details like how many decimals to show.
+            params["mostly_pct"] = ".1f" % (params["mostly"]*100)
+            return [{
+                "template": "Values in $column_A and $column_B must be equal at least $mostly_pct % of the time.",
+                "params": params
+            }]
+
+    @classmethod
+    def expect_table_columns_to_match_ordered_list(cls, expectation):
+        params = substitute_none_for_missing(
+            expectation["kwargs"],
+            ["column_list"]
+        )
+
+        # FIXME: This is slightly wrong, since the whole string (including commas) will get syntax highlighting.
```

Comment:
Is this an argument for a value_list rendering of that expectation instead of bullet list? I realized that bullet vs value was perhaps too much becoming prescriptive vs descriptive.

Reply:
I'm imagining that a bullet list could contain a list of parameters. Again, prototype needed. I'll knock something out.